dagql/idproto/id.proto

```diff
@@ -52,11 +54,24 @@ message ID {
   // handled is a nested list. Don't do that; have a type instead.
   int64 nth = 7;
 
-  // A module that must be loaded prior to evaluating this ID.
+  // The module that provides the implementation of the field.
   //
   // The actual usage of this ID is opaque to the protocol. In Dagger this is
   // the module ID providing the implementation of the field.
-  ID module = 8;
+  Module module = 8;
```

Comment:
I ended up not needing this, but it'll be a big help for visualizing in Cloud, since otherwise there's no easy way to go from an ID to a name.